sdk/maps/azure-maps-search/src/test/java/com/azure/maps/search/MapsSearchClientTest.java

```diff
@@ -0,0 +1,528 @@
+package com.azure.maps.search;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import com.azure.core.exception.HttpResponseException;
+import com.azure.core.http.HttpClient;
+import com.azure.core.models.GeoLineString;
+import com.azure.core.models.GeoObject;
+import com.azure.core.util.Context;
+import com.azure.core.util.polling.SyncPoller;
+import com.azure.maps.search.models.BatchReverseSearchResult;
+import com.azure.maps.search.models.BatchSearchResult;
+import com.azure.maps.search.models.FuzzySearchOptions;
+import com.azure.maps.search.models.LatLong;
+import com.azure.maps.search.models.PointOfInterestCategoryTreeResult;
+import com.azure.maps.search.models.Polygon;
+import com.azure.maps.search.models.ReverseSearchAddressOptions;
+import com.azure.maps.search.models.ReverseSearchAddressResult;
+import com.azure.maps.search.models.ReverseSearchCrossStreetAddressOptions;
+import com.azure.maps.search.models.ReverseSearchCrossStreetAddressResult;
+import com.azure.maps.search.models.SearchAddressOptions;
+import com.azure.maps.search.models.SearchAddressResult;
+import com.azure.maps.search.models.SearchAlongRouteOptions;
+import com.azure.maps.search.models.SearchInsideGeometryOptions;
+import com.azure.maps.search.models.SearchNearbyPointsOfInterestOptions;
+import com.azure.maps.search.models.SearchPointOfInterestCategoryOptions;
+import com.azure.maps.search.models.SearchPointOfInterestOptions;
+import com.azure.maps.search.models.SearchStructuredAddressOptions;
+import com.azure.maps.search.models.StructuredAddress;
+import com.nimbusds.jose.shaded.json.parser.ParseException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+public class MapsSearchClientTest extends MapsSearchClientTestBase {
+
```

Comment:
inconsistent with MapeSearchAsyncClientTest (one extra white line). same discussion about why this needs to be an instance field